AmplifyTestCommon/TestCommonConstants.swift

```diff
@@ -6,6 +6,6 @@
 //
 
 import Foundation
-class TestCommonConstants {
-    static let networkTimeout = TimeInterval(10)
+public class TestCommonConstants {
+    public static let networkTimeout = TimeInterval(10)
```

Comment:
Let us have a auth plugin specific time interval and reduce the time to maybe 2 or 3. The time requirement for test can vary across plugin and it is better to have plugin specific values.

Reply:
Not using this constants for the time being..